data/brands/amenity/restaurant.json

```
@@ -194,6 +194,17 @@
         "name": "Aladdin's Eatery"
       }
     },
+    {
+      "displayName": "L'Antica Pizzeria da Michele",
+      "locationSet": {"include": ["it", "gb-lon.geojson", "gb-north-west.geojson", "es", "jp", "US-CA-Los_Angeles_County.geojson", "ae", "sg", "be", "sa", "de", "nl", "ly", "fr-idf.geojson", "th", "xk", "mt", "ch", "al", "au"]},
```

I'd simplify it use `gb-eng`, and use `us` and `fx`.

```suggestion
      "locationSet": {"include": ["it", "gb-eng", "es", "jp", "us", "ae", "sg", "be", "sa", "de", "nl", "ly", "fx", "th", "xk", "mt", "ch", "al", "au"]},
```